.github/workflows/appimage.yml

```diff
@@ -1,25 +1,31 @@
-name: Build AppImage rplidar module
-
-concurrency: 
-  group: ${{ github.workflow }}-${{ github.ref }}
+name: Build AppImage
 
 on:
   workflow_dispatch:
-  push:
-    branches: [ 'main' ]
-    paths-ignore:
-      - 'README.md'
+    inputs:
+      release_type:
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      release_type:
+        required: true
+        type: string
+    secrets:
+      GCP_CREDENTIALS:
+        required: true
 
 jobs:
   appimage:
-    name: AppImage Build Rplidar Module
+    name: AppImage Build
     strategy:
+      fail-fast: false
```

Comment:
[note] this cahnge as well as the changes on line 25, 28 were copied from rdk. Purpose unknown.

Reply:
fail fast means if ANY job in the matrix fails, stop the other ones. This lets the other ones keep going.

Reply:
Thanks for the clarification!